nemo_text_processing/inverse_text_normalization/hi/graph_utils.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+# Copyright 2024 and onwards Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import string
+from pathlib import Path
+from typing import Dict
+
+import pynini
+from pynini import Far
+from pynini.examples import plurals
+from pynini.export import export
+from pynini.lib import byte, pynutil, utf8
+
+from nemo_text_processing.inverse_text_normalization.hi.utils import get_abs_path, load_labels
+
+NEMO_CHAR = utf8.VALID_UTF8_CHAR
+
+graph_digit = pynini.string_file(get_abs_path("data/numbers/digit.tsv"))
+
+NEMO_HI_DIGIT = pynini.union("०", "१", "२", "३", "४", "५", "६", "७", "८", "९").optimize()
+
+NEMO_HEX = pynini.union(*string.hexdigits).optimize()
```

Comment:
from here below, is this unique to Hindi, or is it the same as in English? It's better not to repeat code, so it would be best to only add here additional variables and functions that are unique to Hindi. Everything else would be imported from en/graph_utils.py

Reply:
Yes, since everything below these lines is the same as in English, would you recommend that I delete everything from line 36 and import the needed elements for the Hindi tagger from en graph_utils?

Reply:
yep!